Editor/UI/JitInspectorView.cs

```diff
@@ -0,0 +1,260 @@
+using Iced.Intel;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using UnityEditor;
+using UnityEditor.UIElements;
+using UnityEngine;
+using UnityEngine.UIElements;
+using Label = UnityEngine.UIElements.Label;
+
+namespace JitInspector.UI
+{
+    using static StringBuilderExtensions;
+    public class JitInspectorView : EditorWindow
+    {
+
+        private static HashSet<string> hashSet = new HashSet<string>();
+        private static List<Assembly> _assemblies;
+
+        [SerializeField]
+        private VisualTreeAsset m_VisualTreeAsset = default;
```

Comment:
nit: is it required that this is called `m_VisualTreeAsset`? It's a bit inconsistent with the naming of the other fields.